pkg/lib/cockpit-components-table.jsx

```diff
@@ -252,7 +260,11 @@ export const ListingTable = ({
                 {showHeader && <Thead>
                     <Tr>
                         {isExpandable && <Th />}
-                        {onSelect && <Th />}
+                        {!onHeaderSelect && onSelect && <Th />}
+                        {onHeaderSelect && onSelect && <Th select={{
+                            onSelect: onHeaderSelect,
+                            isSelected: rows.every(r => r.selected)
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18515-20230316-153310-46e829a6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)